external/leaflet.css

```diff
@@ -464,7 +464,7 @@
 
 /* Default icon URLs */
 .leaflet-default-icon-path {
-    background-image: url(@@INCLUDEIMAGE:images/marker-icon.png@@);
+    background-image: url(""); /* requires Path: not URL! Patched in Boot */
```

Comment:
I suppose that we don't need to patch this at all.
Perhaps it is sort of css fallback, for legacy browsers.

Reply:
Actually `.leaflet-default-icon-path` is a dummy class used for `Icon.Default._detectIconPath`.
We shouldn't include inline image here.